app/views/submission_similarities/show.html.erb

```diff
@@ -26,9 +26,9 @@
   </div>
   <div class="mark_as_suspicious">
     <% if @submission_similarity.status == SubmissionSimilarity::STATUS_NOT_PLAGIARISM %>
-      <p><%= link_to "Mark these similarities as suspicious", assignment_submission_similarity_suspect_as_plagiarism_url(@assignment, @submission_similarity), :method => :put, data: {confirm: "Mark these similarities as suspicious?"} %> or <%= link_to "Confirm these similarities as plagiarism", assignment_submission_similarity_confirm_as_plagiarism_url(@assignment, @submission_similarity), :method => :put, data: {confirm: "Confirm these similarities as plagiarism?"} %></p>
+      <%= button_to "Confirm these similarities as plagiarism", assignment_submission_similarity_confirm_as_plagiarism_url(@assignment, @submission_similarity), :method => :put, data: {confirm: "Confirm these similarities as plagiarism?"} %> <%= link_to "or mark these similarities as suspicious...", assignment_submission_similarity_suspect_as_plagiarism_url(@assignment, @submission_similarity), :method => :put, data: {confirm: "Mark these similarities as suspicious?"} %> 
```

Comment:
Important: These should show up as buttons and not links because they are processes/actions and not views.

Reply:
PR#92 (https://github.com/WING-NUS/SSID/pull/92) has fixed it.